src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -105,13 +106,39 @@
 							v-html="heroSubHeadline"
 							class="tw-mb-2 md:tw-mb-3"
 						></h2>
-						<div v-if="heroBody" class="tw-prose tw-mb-2 md:tw-mb-3" ref="heroBodyCopy">
+						<div
+							v-if="heroBody" class="tw-prose tw-mb-2 md:tw-mb-3"
+							ref="heroBodyCopy"
+						>
 							<dynamic-rich-text :html="heroBody" />
 						</div>
-						<button-wrapper
-							class="tw-w-full md:tw-w-auto"
-							:content="buttonContent"
-						/>
+						<div
+							v-if="gridHeroBody.length"
+							:class="{'tw-grid md:tw-grid-cols-2 md:tw-grid-rows-2 tw-gap-1 md:tw-gap-3'
+								: gridHeroBody.length }"
+						>
+							<template v-if="gridHeroBody.length">
```

Comment:
The `v-if` on this template shouldn't be needed as the same check is on the div above.